vars/params.env

```diff
@@ -1,4 +1,5 @@
 loggingElasticSearchImage=docker.elastic.co/elasticsearch/elasticsearch:7.2.0
 loggingVolumeStorage=1Gi
 artifactRepositoryBucket=mlpipeline.onepanel.io
-artifactRepositoryEndpoint=minio-service.onepanel:9000
\ No newline at end of file
+artifactRepositoryEndpoint=minio-service.onepanel:9000
+defaultNamespace=bees
```

Comment:
@aleksandrmelnikov I think you forgot to revert the value for this back to `default`.